content/en/docs/concepts/services-networking/topology-aware-routing.md

```diff
@@ -15,45 +15,68 @@ description: >-
-consumers of EndpointSlice (or Endpoints) objects, so that traffic to
-those network endpoints can be routed closer to where it originated.
+{{< note >}}
+Prior to Kubernetes 1.27, this feature was known as _Topology Aware Hints_.
+{{</ note >}}
 
-For example, you can route traffic within a locality to reduce
-costs, or to improve network performance.
+_Topology Aware Routing_ adjusts routing behavior to prefer keeping traffic in
+the zone it originated from. In some cases this can help reduce costs or improve
+network performance.
 
 <!-- body -->
 
 ## Motivation
 
 Kubernetes clusters are increasingly deployed in multi-zone environments.
-_Topology Aware Hints_ provides a mechanism to help keep traffic within the zone
-it originated from. This concept is commonly referred to as "Topology Aware
-Routing". When calculating the endpoints for a {{< glossary_tooltip term_id="Service" >}},
-the EndpointSlice controller considers the topology (region and zone) of each endpoint
-and populates the hints field to allocate it to a zone.
-Cluster components such as the {{< glossary_tooltip term_id="kube-proxy" text="kube-proxy" >}}
-can then consume those hints, and use them to influence how the traffic is routed
-(favoring topologically closer endpoints).
+_Topology Aware Routing_ provides a mechanism to help keep traffic within the
+zone it originated from. When calculating the endpoints for a {{<
+glossary_tooltip term_id="Service" >}}, the EndpointSlice controller considers
+the topology (region and zone) of each endpoint and populates the hints field to
+allocate it to a zone. Cluster components such as {{< glossary_tooltip
+term_id="kube-proxy" text="kube-proxy" >}} can then consume those hints, and use
+them to influence how the traffic is routed (favoring topologically closer
+endpoints).
 
-## Using Topology Aware Hints
+## Enabling Topology Aware Routing
 
-You can activate Topology Aware Hints for a Service by setting the
-`service.kubernetes.io/topology-aware-hints` annotation to `auto`. This tells
-the EndpointSlice controller to set topology hints if it is deemed safe.
-Importantly, this does not guarantee that hints will always be set.
+{{< note >}}
+Prior to Kubernetes 1.27, this behavior was controlled using the
+`service.kubernetes.io/topology-aware-hints` annotation.
+{{</ note >}}
 
-## How it works {#implementation}
+You can enable Topology Aware Routing for a Service by setting the
+`service.kubernetes.io/topology-mode` annotation to `Auto`. When there are
+enough endpoints available in each zone, Topology Hints will be populated on
+EndpointSlices to allocate individual endpoints to specific zones, resulting in
+traffic being routed closer to where it originated from.
 
-The functionality enabling this feature is split into two components: The
-EndpointSlice controller and the kube-proxy. This section provides a high level overview
-of how each component implements this feature.
+### When it works best
```

Comment:
Is intended as a subheading of “Enabling Topology Aware Routing?”

Reply:
Updated, thanks!